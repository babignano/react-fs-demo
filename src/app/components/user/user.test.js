import React from 'react';
import User from './user';
import renderer from 'react-test-renderer';

describe('User', () => {
    test('should match snapshot', () => {
        const component = renderer.create(<User />);
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});
