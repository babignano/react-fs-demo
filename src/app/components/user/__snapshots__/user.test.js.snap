// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`User should match snapshot 1`] = `
<h1>
  User
</h1>
`;
